contracts/universalSchemes/SchemeRegistrar.sol

```diff
@@ -35,7 +35,7 @@ contract SchemeRegistrar is UniversalScheme {
     }
 
     // A mapping from the organization (Avatar) address to the saved data of the organization:
-    mapping(address=>mapping(bytes32=>SchemeProposal)) public organizationsProposals;
+    mapping(address=>mapping(bytes32=>SchemeProposal)) public organizationProposals;
```

Comment:
please revert it back to organizationsProposals

Reply:
I'm not sure where this diff is coming from, it is named `organizationProposals` in the code.

Reply:
it should be name organizationsProposals